scripts/legislation/africa/lesotho_english.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+function remove_all_text_before_first_header {
+  sed -n '/^\Enacted by the Parliament of Lesotho\./,$p' | \
+    sed -n '/^PART I/,$p' 
+}
+
+function prefix_article_numbers_with_article_literal {
+  sed -E 's/^([0-9]+\.)/Article \1/'
+}
+
+function replace_parentheses_around_article_delimiters_with_square_brackets {
+  sed -E 's/^\(([A-Za-z0-9]+)\)/[\1]/'
+}
+
+function move_article_titles_after_article_numbers {
+  sed -En '${p;q};N;/\nArticle/{s/^(.*)\n(Article [0-9]+\.) /\2 \1|/p;b};P;D'
+}
+
+function move_article_titles_above_articles {
+  sed -E 's/^(\([0-9]+\) )([^|]+)\|/\2\n\1/'
+}
+
+function remove_all_text_after_last_article {
+  sed -E '/^\(116\)/q'
```

Comment:
Please replace this with the following:

```
sed -E 's/^\(116\)/,${/^\(116\)/!d}'
```

Reply:
This line of code returns an error: 

> sed: -e expression #1, char 16: unknown option to `s'

Reply:
Oops. Yes, the `s` should not have been part of my suggestion. Sorry about that.